witness/src/Dscp/Witness/Web/Types.hs

```diff
@@ -152,6 +152,9 @@ instance Buildable (ForResponseLog HashIs) where
 instance Buildable (ForResponseLog TxId) where
     build = buildForResponse
 
+instance Buildable (ForResponseLog FairCVCheckResult) where
+    build (ForResponseLog res) = build res
+
```

Comment:
@Martoon-00 Since the responses are just bool (until DSCP-404), let's just print the number of failures and number of successes here. In DSCP-404, we may elaborate on that.

Reply:
Noice :+1: